### ISSUE TITLE
NUM tokens not showing up in CaptureClub

### ISSUE THREAD

**Issue by JAVIERCE31** [numbersprotocol/community-support#251]
Hello,
since the update my tokens don't appear in CaptureClub, which prevents me from being able to continue publishing my work (In App Capture if my tokens are shown).

Lately I had a good sales rate, but since the update I can no longer upload any of my photos and I have stopped having sales.

Expected behavior
Show my tokens to be able to publish my photos

Screenshots
![image](https://user-images.githubusercontent.com/119164748/221527499-5f3f8bad-378d-4991-b010-d3312c824913.png)

![image](https://user-images.githubusercontent.com/119164748/221527556-36c939da-c83f-4215-9bf1-779fdfeb1fb2.png)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1203248692432071/1204064239742675) by [Unito](https://www.unito.io)

**Comment by JAVIERCE31** [comment]
Hi, it's been a long time since I reported this bug to you on CaptureClub.
My photos have ceased to have visibility and I already give up my sales. This was the only way to recover a bit from the 90% loss in the value of my tokens.
I just hope you surprise me with a big update.
All the best.

**Comment by kennyhungQA** [comment]
Hi, @JAVIERCE31 

We apologize for any inconvenience caused. Currently, the functionality of CaptureClub has been replaced by the [Capture App.](https://link.numbersprotocol.io/CaptureApp) The latest version(0.77.2) of the Capture App allows you to upload your existing creations and continue with the Mint & Share network action.

Regarding the camera crash issue you mentioned in another ticket, our product team is actively working on resolving it. Once the final testing is complete and it's approved by Google Play/App Store, we will reply to this ticket and kindly request that you update to the latest version of the Capture App.

We appreciate your understanding and patience. If you have any further questions or need further assistance, please feel free to let us know.

Best regards,
Kenny Product Team
